src/msw/listctrl.cpp

```diff
@@ -334,6 +334,11 @@ void wxListCtrl::MSWSetExListStyles()
         // it seems better to enable it by default than disable
         LVS_EX_HEADERDRAGDROP
     );
+
+    // If using the default background style with double buffering enabled then switch to system
+    // Using wxBG_STYLE_ERASE while will cause flicker despite double buffering
```

Comment:
Yeah that definitely broke at least panels and statusbars. Static text seemed to be ok with it though. I didn't try all controls since that was enough to make me not want to do it. I think the safer option is to just have a static event table entry for these two controls specifically. That way it's the last handler to get called still allowing any other custom erase background logic to happen before this.